bin/metric-postgres-statsdb.rb

```diff
@@ -62,9 +62,17 @@ class PostgresStatsDBMetrics < Sensu::Plugin::Metric::CLI::Graphite
          long: '--port PORT'
 
   option :database,
-         description: 'Database name',
+         description: 'Database to connect on',
          short: '-d DB',
-         long: '--db DB'
+         long: '--db DB',
+         default: 'postgres'
+
+  option :all_databases,
+         description: 'Get stats for all the databases instead of only the one we are connected on.',
```

Comment:
Highly subjective but I would change this to `'Get stats for all databases'`

Reply:
agreed.

Reply:
```suggestion
         description: 'Get stats for all the databases',
```